packages/aztec.js/src/note/index.js

```diff
@@ -20,18 +27,24 @@ class Note {
      *
      * @param {string} publicKey hex-formatted public key
      * @param {string} viewingKey hex-formatted viewing key
+     * @param {Object} access mapping between an Ethereum address and the linked publickey
```

Comment:
Not an object. Or at least not strictly an object.

Reply:
updated